src/org/usfirst/frc4904/robot/commands/IndexerUnload.java

```diff
@@ -0,0 +1,12 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.subsystems.Shooter;
+import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
+
+public class IndexerUnload extends MotorConstant {
+	public IndexerUnload() {
+		super(RobotMap.Component.shooter.indexer, Shooter.INDEXER_UNLOAD_SPEED);
```

Comment:
Ah, good point - the flywheel should only be accessed through the shooter. The reason I made this mistake is because the flywheel is its own subsystem, while the indexer's just a motor.